neptune-core/src/application/json_rpc/core/api/ops.rs

```diff
@@ -53,6 +56,21 @@ pub enum RpcMethods {
 
     #[namespace(Namespace::Chain)]
     TipAnnouncements,
+
+    #[namespace(Namespace::Chain)]
+    BlockInfo,
+
+    #[namespace(Namespace::Chain)]
+    BlockDigest,
+
+    #[namespace(Namespace::Chain)]
+    BlockDigestsByHeight,
```

Comment:
Belongs in archive, not in `Chain`, since `Chain` can only handle information about the tip (i.e. the latest block).